transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -137,31 +137,22 @@ public void openFileDescriptors() {
             success = true;
         } finally {
             if (!success) {
-                if (epollFd != null) {
-                    try {
-                        epollFd.close();
-                    } catch (Exception e) {
-                        // ignore
-                    }
-                }
-                if (eventFd != null) {
-                    try {
-                        eventFd.close();
-                    } catch (Exception e) {
-                        // ignore
-                    }
-                }
-                if (timerFd != null) {
-                    try {
-                        timerFd.close();
-                    } catch (Exception e) {
-                        // ignore
-                    }
-                }
+                closeFileDescriptor(epollFd);
+                closeFileDescriptor(eventFd);
+                closeFileDescriptor(timerFd);
             }
         }
     }
 
+    private static void closeFileDescriptor(FileDescriptor fd) {
+        if (fd != null) {
+            try {
+                fd.close();
+            } catch (Exception e) {
+                // ignore
+            }
+        }
+    }
```

Comment:
Add a line break between this and the next method.

Reply:
ok，I made it up in the second commit

Reply:
Also between this one and `newTaskQueue` below.

Reply:
sorry，my carelessness